reportportal/values.yaml

```diff
@@ -157,6 +157,9 @@ minio:
   region:
   accesskey: <minio-accesskey>
   secretkey: <minio-secretkey>
+  bucketPrefix: ""
+  defaultBucketName: ""
+  integrationSaltPath: "keystore"
```

Comment:
Default left as empty "" is fine as the value is checked in the templates.

Reply:
set to `""`.

Reply:
@waynesun09 fixed